app/main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command = command.split()
+    initial_file, destination = command[1], command[2]
```

Comment:
You can do this like: 
```suggestion
    command, initial_file, destination = command.split()
```